relay-server/src/services/buffer/envelope_stack/sqlite.rs

```diff
@@ -228,6 +234,28 @@ impl EnvelopeStack for SqliteEnvelopeStack {
     }
 }
 
+impl Evictable for SqliteEnvelopeStack {
+    async fn evict(&mut self) {
```

Comment:
I wanted eviction to be an infallible operation, since in case of errors, we can't do much from Relay's standpoint.